jobs/etl_jobs/internal/export_clickhouse/schema/analytics/monthly_aggregated_venue_collective_revenue.sql

```diff
@@ -1,23 +1,22 @@
 CREATE OR REPLACE TABLE analytics.monthly_aggregated_venue_collective_revenue ON cluster default
-    ENGINE = SummingMergeTree()
-    PARTITION BY month
-    ORDER BY (venue_id)
-    SETTINGS storage_policy='gcs_main'
+ENGINE = SummingMergeTree()
+PARTITION BY month
+ORDER BY (venue_id)
+SETTINGS storage_policy='gcs_main'
 AS
 SELECT
-    date_trunc('MONTH', coalesce(toDate (used_date), toDate (creation_date))) AS month,
-    cast(venue_id as String) as venue_id,
+    toStartOfMonth(coalesce(toDate(used_date), toDate(creation_date))) AS month,
+    cast(venue_id AS String) AS venue_id,
     sum(
-        case
-            when collective_booking_status = 'USED'
-            or collective_booking_status = 'REIMBURSED' then booking_amount
-            else 0
-        end
+        CASE
+            WHEN collective_booking_status IN ('USED', 'REIMBURSED') THEN booking_amount
+            ELSE 0
+        END
     ) AS revenue,
     sum(booking_amount) AS expected_revenue
 FROM
     intermediate.collective_booking
 WHERE
     venue_id IS NOT NULL
 GROUP BY
-    1,2
+    month, venue_id;
```

Comment:
alors on s'en fiche dans ce hotfix, mais [dbt recommande de faire des group by 1,2,3](https://docs.getdbt.com/best-practices/how-we-style/2-how-we-style-our-sql) (à ma surprise) plutôt que par  nom 
![image](https://github.com/user-attachments/assets/9097299c-9cc8-44f9-ae76-1a33abd41dd3)